projects/cm_mcu/MonitorI2CTask.h

```diff
@@ -0,0 +1,90 @@
+/*
+ * MonitorI2CTask.h
+ *
+ *  Created on: June 30, 2022
+ *      Author: pkotamnives
+ */
+
+#ifndef PROJECTS_CM_MCU_MONITORI2CTASK_H_
+#define PROJECTS_CM_MCU_MONITORI2CTASK_H_
+
+#include "common/smbus.h"
+#include "semphr.h"
+#include "Tasks.h"
+
+extern SemaphoreHandle_t xMonSem;
+
+// pilfered and adapted from http://billauer.co.il/blog/2018/01/c-pmbus-xilinx-fpga-kc705/
+// enum sm_type { SM_VOLTAGE, SM_NONVOLTAGE, SM_STATUS, SM_LINEAR11, SM_LINEAR16U, SM_LINEAR16S } ;
+enum sm_type { SM_STATUS,
```

Comment:
I think this enum largely duplicates a similar one elsewhere, is there a reason to duplicate?

Reply:
I was trying to imitate the MonitorTask code at the beginning but I don't think it is necessary in the code now. I'll remove this enum and this member in struct then?

Reply:
or you could move the enum to e.g. Tasks.h and use one enum in both places?

Reply:
Okay, I can do that. Done.